oneanddone/tasks/views.py

```diff
@@ -89,11 +89,15 @@ def form_valid(self, form):
         task_name = feedback.attempt.task.name
         subject = 'Feedback on %s from One and Done' % task_name
         task_link = 'http'
+        feedback_link = 'http'
         if self.request.is_secure():
             task_link += 's'
+            feedback_link += 's'
```

Comment:
You are duplicating for `feedback_link` what is already being done for `task_link`. Instead of having two copies of the same code, you could just call the variable `link_prefix`, and then use that as the start of both `task_link` and `feedback_link` below.